docs/sdks/client-sdks/flutter/quickstart.mdx

```diff
@@ -0,0 +1,293 @@
+
+```dart
+/// Represents an assignment event.
+class AssignmentEvent {
+  /// The allocation the subject was assigned to
+  final String? allocation;
+
+  /// The experiment identifier
+  final String? experiment;
+
+  /// The feature flag identifier
+  final String featureFlag;
+
+  /// The format of the assignment
+  final String format;
+
+  /// The variation key that was assigned
+  final String? variation;
+
+  /// The subject identifier
+  final String subject;
+
+  /// The timestamp of the assignment
+  final String timestamp;
+
+  /// The subject attributes
+  final Map<String, dynamic>? subjectAttributes;
+
+  /// Additional metadata
+  final Map<String, dynamic>? metaData;
+}
+
+// Implement your own `AssignmentLogger`
+class YourAssignmentLogger implements AssignmentLogger {
+  @override
+  void logAssignment(AssignmentEvent event) {
+    // Your logging logic here
+  }
+}
+
+await Eppo.initialize(
+  PrecomputeArguments(subject: subject),
+  SdkOptions(
+    sdkKey: sdkKey,
+    assignmentLogger: YourAssignmentLogger(),
+  ),
+);
+```
+
+:::note
+For your production application, we have documentation on how to set up logging with multiple popular data warehouses and 
+logging systems in the [Assignment page](/sdks/client-sdks/flutter-web/assignments/#logging-data-to-your-data-warehouse).
+:::
+
+## Contextual Bandits
+
+For bandits, a separate logger handles its logging:
+
+```dart
+/// Represents a bandit event.
+class BanditEvent {
+  /// The timestamp when the event occurred
+  final String timestamp;
+
+  /// The feature flag key
+  final String featureFlag;
+
+  /// The bandit key
+  final String bandit;
+
+  /// The subject key
+  final String subject;
+
+  /// The action associated with the bandit
+  final String? action;
+
+  /// The probability of the action
+  final double? actionProbability;
+
+  /// The optimality gap
+  final double? optimalityGap;
+
+  /// The model version
+  final String? modelVersion;
+
+  /// Numeric attributes of the subject
+  final Map<String, double>? subjectNumericAttributes;
+
+  /// Categorical attributes of the subject
+  final Map<String, String>? subjectCategoricalAttributes;
+
+  /// Numeric attributes of the action
+  final Map<String, double>? actionNumericAttributes;
+
+  /// Categorical attributes of the action
+  final Map<String, String>? actionCategoricalAttributes;
+
+  /// Metadata about the SDK
+  final Map<String, dynamic> metaData;
+}
+
+// Implement your own `BanditLogger`
+class YourBanditLogger implements BanditLogger {
+  @override
+  void logBanditEvent(BanditEvent event) {
+    // Your logging logic here
+  }
+}
+
+await Eppo.initialize(
+  PrecomputeArguments(
+    subject: subject,
+    banditActions: {
+      'flag-with-shoe-bandit': {
+        'nike': {
+          'brand_affinity': 0.4,
+          'from': 'usa',
+        },
+        'adidas': {
+          'brand_affinity': 2.0,
+          'from': 'germany',
+        },
+      },
+    },
+  ),
```

Comment:
I think the bandit actions API needs more attention - I will review current implementation as this might need to be a list.

Reply:
Double-checked, matches up with javascript: https://github.com/Eppo-exp/js-sdk-common/blob/main/src/interfaces.ts#L199C1-L203C2

Tricky to test as our QA bandits are down.